src/pages/PrivacyPolicy/PrivacyPolicy.test.js

```diff
@@ -1,19 +1,34 @@
 import React from "react"
-import Enzyme, { shallow } from "enzyme"
-import Adapter from "enzyme-adapter-react-16"
+import "@testing-library/jest-dom"
+import { render, screen } from "@testing-library/react"
 import { PrivacyPolicy } from "./PrivacyPolicy"
-
-Enzyme.configure({ adapter: new Adapter() })
+import { BrowserRouter } from "react-router-dom"
 
 describe("<PrivacyPolicy />", () => {
-  it("renders a header", () => {
-    const contributor = shallow(<PrivacyPolicy />)
-    const contributorHeading = contributor.find("Header")
-    expect(contributorHeading.length).toEqual(1)
+  beforeEach(() => {
+    render(
+      <BrowserRouter>
+        <PrivacyPolicy />
+      </BrowserRouter>
+    )
   })
-  it("renders a footer", () => {
-    const contributor = shallow(<PrivacyPolicy />)
-    const contributorFooter = contributor.find("Footer")
-    expect(contributorFooter.length).toEqual(1)
+
+  it("renders without crashing", () => {
+  
```

Comment:
Can you remove this test? I feel like it is just getting in the way. And you have better tests below.